### PR TITLE
Add docs for PR process.

### LINKED ISSUE #5685: Add documentation for Pull request process
What i have in mind is something similar to https://docs.pulpproject.org/pulpcore/contributing/pull-request-walkthrough.html

Specifically, I don't see in our docs that we require 2 approvals to merge.

### PR DESCRIPTION
**Pull request by jberkhahn**
Fixes #5685

Hopefully my markdown is right.

### CODE REVIEW

File: website/content/en/docs/contribution-guidelines/opening-pull-requests.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+1. Push your commit to your fork and [open a pull request][gh-fork-pr].
+
+
+## Review
+
+Before a pull request can be merged, tests must pass in CI and it must be reviewed. A PR
+must be approved by 2 reviewers, one of which must be at least at least a reviewer and one
+of which must be at least an approver, per the [Operator Framework community guidelines][of-contributor-ladder].
+
+Please feel free to the developers to get eyes on your PR, whether through @'ing on the PR itself,
```
Comment (everettraven) [inline website/content/en/docs/contribution-guidelines/opening-pull-requests.md:24]: Typo?:
```suggestion
Please feel free to message the developers to get eyes on your PR, whether through @'ing on the PR itself,
```

**Review by everettraven — APPROVED** [review]
Other than one typo this looks good to me!

**Comment by jberkhahn** [comment]
fixed the typo

**Review by everettraven — APPROVED** [review]
/lgtm

**Review by rashmigottipati — APPROVED** [review]
/lgtm

**Pull request merged**